ckanext/switzerland/commands.py

```diff
@@ -18,13 +17,18 @@ class OgdchCommand(ckan.lib.cli.CkanCommand):
         # Cleanup datastore
         paster ogdch cleanup_datastore
         # Cleanup harvester jobs and objects:
-        # - deletes the all harvest jobs and objects except the latest
-        # - how many jobs to keep can be set in the ckan config
-        paster ogdch cleanup_harvesters [{source_id}]
+        # - deletes the all harvest jobs and objects except the latest n
+        # - the default number of jobs to keep is 10
+        paster ogdch cleanup_harvesters [{source_id}] [--keep={n}}]
     '''
     summary = __doc__.split('\n')[0]
     usage = __doc__
 
+    def __init__(self, name):
+        super(ckan.lib.cli.CkanCommand, self).__init__(name)
+        self.parser.add_option('--keep', action="store", type="int", dest='nr_of_jobs_to_keep',
+                               default=10, help='The number of jobs to keep when clearing harvesters')
```

Comment:
Clearing harvesters might be little misleading. Can you change it to something that makes it clear what is cleaned/cleared to not confuse it with the paster command that clears the harvest-source?

Reply:
Done